visual_behavior/data_access/reformat.py

```diff
@@ -258,7 +258,7 @@ def add_trial_type_to_trials_table(trials):
 
 def add_reward_rate_to_trials_table(trials):
     trials['rewarded'] = [1 if np.isnan(reward_time) == False else 0 for reward_time in trials.reward_time.values]
-    trials['reward_rate'] = trials['rewarded'].rolling(window=100, min_periods=1, win_type='triang').mean()
+    trials['reward_rate'] = trials['rewarded'].rolling(window=100, min_periods=1, win_type='triang').mean() * (60 / .75)  # units of rewards/min
```

Comment:
I don't think this is correct. Each row of the trials table is a trial, which has variable length. To convert the entries of the trials table into a reward rate you would need to use the time information of the rewards. This approach works for the stimulus presentation table because the duration between stimuli is fixed (mostly).

Reply:
I changed this to use the reward_rate calculated using stimulus_presentations instead of calculating reward rate on trials itself in the function` reformat.add_reward_rate_and_engagement_state_to_trials_table()`

Reply:
Should you remove the function `add_reward_rate_to_trials_table` then?